libs/Wallet/include/Status/Wallet/WalletController.h

```diff
@@ -38,6 +38,7 @@ class WalletController: public QObject
     /// \todo consider if complex approach of keeping ownership here and enforcing a unique instance
     ///       or not reusing the account list and make it singleton are better options
     Q_INVOKABLE [[nodiscard]] Status::Wallet::NewWalletAccountController* createNewWalletAccountController() const;
+    Q_INVOKABLE [[nodiscard]] Status::Wallet::SavedAddressesController* createSavedAddressesController() const;
```

Comment:
I find that adding a comment to clarify that the caller takes ownership of the object helps.

Reply:
@stefandunca Maybe it would be clearer, on the other hand it's default behavior ([QQmlEngine::setObjectOwnership](https://doc.qt.io/qt-6/qjsengine.html#setObjectOwnership)() is not called to change it)

Reply:
I agree that is a QML convention however if used in tests, later on, the implementation should behave like QML. Maybe a note at the top will be better than specifying this for each `Q_INVOKABLE` function instead of repeating "/// \note caller (QML) takes ownership of the returned object" for every call.